### PR TITLE
Az.MonitoringSolutions

### PR DESCRIPTION
**Pull request by isra-fel**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/633
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/MonitoringSolutions/docs/Get-AzMonitorLogAnalyticsSolution.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+external help file:
+Module Name: Az.MonitoringSolutions
+online version: https://docs.microsoft.com/en-us/powershell/module/az.monitoringsolutions/get-azmonitorloganalyticssolution
+schema: 2.0.0
+---
+
+# Get-AzMonitorLogAnalyticsSolution
+
+## SYNOPSIS
+Retrieves the user solution.
```
Comment (isra-fel) [inline src/MonitoringSolutions/docs/Get-AzMonitorLogAnalyticsSolution.md:11]: What's the best way to update this?

File: src/MonitoringSolutions/docs/Update-AzMonitorLogAnalyticsSolution.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+external help file:
+Module Name: Az.MonitoringSolutions
+online version: https://docs.microsoft.com/en-us/powershell/module/az.monitoringsolutions/update-azmonitorloganalyticssolution
+schema: 2.0.0
+---
+
+# Update-AzMonitorLogAnalyticsSolution
+
+## SYNOPSIS
+Patch a Solution.
```
Comment (isra-fel) [inline src/MonitoringSolutions/docs/Update-AzMonitorLogAnalyticsSolution.md:11]: Want this to be updated too

**Pull request merged**